app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,58 @@
+<section class="container">
+  <h1 class="text-center mb-5"><%= @recipe.name %></h1>
+  <span class="badge badge-primary">Private</span>
+  <div class="recipe-info d-flex justify-content-around">
+    <div>
+      <h5>Preparation time: <%= @recipe.preparation_time %></h5>
+      <h5>Cooking time: <%= @recipe.cooking_time %></h5>
+      <div>
+        <h3 class="mt-5">Description</h3>
+        <p><%= @recipe.description %></p>
+      </div>
+    </div>
+    <div class="recipe-stat">
+      <% if @recipe.public %>
+        <span class="btn btn-warning">Public</span>
+      <% else %>
+        <span class="btn btn-primary">Private</span>
+      <% end %>
+    </div>
+  </div>
+  <% if can? :destroy, @recipe %>
+  <div class="mt-5 d-flex justify-content-around">
+    <%= link_to "Generate List", shopping_list_index_path(recipe: @recipe), class: "btn btn-danger" %>
+    <%= link_to "Add ingredient", new_user_recipe_recipe_food_path(@recipe.user, @recipe), class: "btn btn-danger" %>
+  </div>
+  <table class="table table-striped table-bordered mt-5 text-center">
+    <thead>
+      <tr>
+        <th scope="col">Food</th>
+        <th scope="col">Measurement unit</th>
+        <th scope="col">Price</th>
+        <th scope="col">Actions</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% if @recipe.recipe_foods.length.positive? %>
+        <% @recipe.recipe_foods.each do |recipe_food|%>
+          <tr>
+            <td> <%= recipe_food.food.name%></td>
+            <td> <%= recipe_food.quantity%> <%= recipe_food.food.measurement_unit %></td>
+            <td> $ <%= recipe_food.food.price%></td>
+            <td>
+              <%= button_to 'delete', recipe_food_path(recipe_food.id), method: :delete, class:"btn btn-warning"%>
+            </td>
+          </tr>
```

Comment:
- [ ] I was expecting to see the foods I add on a recipe show page, but none was shown.

![my-food](https://user-images.githubusercontent.com/53124548/201414461-3d98f1b7-dce4-4f93-8d32-76f675aa85a6.png)

- [ ] The option to add an ingredient is not well set, a user is given a number field to enter a number, is that number a quantity or amount?